CE.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
What's this file? I've not seen this before and it's something that will show up in the top-level directory, which we try to avoid cluttering up.